### PR TITLE
removed inconsistencies

### PR DESCRIPTION
**Pull request by petervanderstok**
in particular: resource naming, group creation, changelog

### CODE REVIEW

File: draft-ietf-core-resource-directory.md [hunk 1]
```diff
@@ -976,26 +980,25 @@ Plurality of link references exists if, and only if, two or more links in a Reso
...
 
-TODO: suggestion to do a GET /rd to obtain a list of registrations loc, ep,...
 
 ### Registration Update {#update}
 
 The update interface is used by an endpoint to refresh or update its
 registration with an RD. To use the interface, the endpoint sends a POST request to the registration resource returned in the Location header option in the response returned from the initial registration operation. The POST request (in contrast to PUT) allows replacing a selection of a resource.
 
-An update MAY update the lifetime or context registration parameters
+An update MAY update the lifetime- or the context- registration parameters
```
Comment (chrysn) [inline draft-ietf-core-resource-directory.md]: We haven't typeset them as lifetime-registration parameters or similar; 'update the "lifetime" or the "context" registration parameters' perhaps?

File: draft-ietf-core-resource-directory.md [hunk 2]
```diff
@@ -976,26 +980,25 @@ Plurality of link references exists if, and only if, two or more links in a Reso
 contain identical context, target, and relation values. This condition would be likely to arise if there were multiple co-ordinators or configuration tools, each with a different
 set of configuration values for the same resource.
 
-A Resource Directory SHOULD reject a registration, or an operation on a registration, which would result in a plurality of link references within the the context of the Resource Directory. There is no requirement in this document for a resource directory to check for plurality of reference. Resource Directory operations which are rejected due to reference plurality SHOULD be returned the "Conflict" code, indicating that there is someting wrong with the request.
+A Resource Directory SHOULD reject a registration, or an operation on a registration resource, which would result in a plurality of link references within the directory resource. There is no requirement in this document for a resource directory to check for plurality of reference. Resource Directory operations which are rejected due to reference plurality SHOULD return the "Conflict" code, indicating that there is someting wrong with the request.
```
Comment (chrysn) [inline draft-ietf-core-resource-directory.md]: See https://github.com/core-wg/resource-directory/issues/70

**Comment by petervanderstok** [comment]
> -At the first registration of a set of entries, a "registration
> resource" is created,
> +At the first registration of a set of entries, a "directory resource"
> is created,
> 

That's correct. an directory resource is created, unless already 
present,
such that the registration resources can be created.

Other text wanted?

chrysn schreef op 2017-10-24 15:15:

**Comment by chrysn** [comment]
I'd just leave the line in question as it was.

**Pull request merged**